src/xunit.performance.api/EtwPerformanceMetricEvaluationContext.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Parsers.Kernel;
+using Microsoft.Diagnostics.Tracing.Parsers.MicrosoftXunitBenchmark;
+using Microsoft.Xunit.Performance.Sdk;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal class EtwPerformanceMetricEvaluationContext : PerformanceMetricEvaluationContext, IDisposable, IPerformanceMetricReader
```

Comment:
Let's see if we can deprecate the older stuff soon, and maybe have a branch that has the deprecated stuff in it so that if we have to hotfix it we can.  Then, we can delete the old stuff out of master, and not even have to worry about it.

---
In reply to: [102346911](https://github.com/Microsoft/xunit-performance/pull/191#discussion_r102346911) [](ancestors = 102346911)